jobserver/management/commands/link_workspace_to_project.py

```
@@ -0,0 +1,36 @@
+import sys
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from furl import furl
+
+from jobserver.models import Project, Workspace
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument("project")
+        parser.add_argument("workspace")
+
+    def handle(self, *args, **options):
+        project_slug = options["project"]
+        try:
+            project = Project.objects.get(slug=project_slug)
+        except Project.DoesNotExist:
+            self.stderr.write(f"Unknown Project slug: {project_slug}")
+            sys.exit(1)
```

Very disciplined of you! I would've just let the exception bubble up here